### PR TITLE
Release Godot 4.1.4 RC1 and Godot 4.2.2 RC1, and publish release notes

### PR DESCRIPTION
**Pull request by YuriSizov**
A new format, may be a bit awkward 😳

### CODE REVIEW

File: collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-1.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: "Release candidate: Godot 4.1.4 RC 1 & 4.2.2 RC 1"
```
Comment (JohnVeness) [inline collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-1.md]:
```suggestion
title: "Release candidates: Godot 4.1.4 RC 1 & 4.2.2 RC 1"
```

**Review by JohnVeness — COMMENTED** [review]
Just one correction I noticed.

I can see merits in combining both these releases into one post, but on balance I think it might be slightly clearer to have two separate posts...

File: collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-1.md (hunk 1 above)
Comment (YuriSizov) [reply collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-1.md]: This was deliberate, but I can change it.

File: collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-1.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+- If you have been experiencing issues opening large tilemaps due to preview generation, this release addresses those issues ([GH-87470](https://github.com/godotengine/godot/pull/87470)).
+
+- Several visualizations in the 3D editor have been improved, including origin lines ([GH-83895](https://github.com/godotengine/godot/pull/83895)) and `Curve3D`-related debug information ([GH-83698](https://github.com/godotengine/godot/pull/83698)). Origin lines should appear thicker and without aliasing now!
+
+- Multiple animation issues have been addressed following the changes introduced in Godot 4.2 with the `AnimationMixer` node ([GH-86046](https://github.com/godotengine/godot/pull/86046), [GH-86221](https://github.com/godotengine/godot/pull/86221), [GH-86227](https://github.com/godotengine/godot/pull/86227), [GH-86718](https://github.com/godotengine/godot/pull/86718), and others).
+
+- A new `PackedRealArray` type is now available for module developers who also target GDExtension ([GH-86324](https://github.com/godotengine/godot/pull/86324)).
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-4-and-4-2-2-rc-1.md]:
```suggestion
- A new `PackedRealArray` type alias is now available for module developers who also target GDExtension ([GH-86324](https://github.com/godotengine/godot/pull/86324)).
```
Makes it clearer we're not adding a whole new Variant type of bloat just for this.

**Review by akien-mga — APPROVED** [review]
Looks good to me overall.

I'm also a bit on the fence regarding the new format, it feels a bit awkward to have two releases together. I think it would also be fine to have two blog posts released at the same time, one for each release, and cross-reference each other in an intro which can pretty much be copy-pasted.

But I'm also fine trialing this format, and see if users are confused or not.

**Comment by YuriSizov** [comment]
I think I could've done it better. With some adjustments to the intro section, I could've made one big section about one release and one about another, so highlights and downloads are next to each other. Downloads feel messy to me, personally, but otherwise I think it's okay.

But we'll see, I guess, and we'll make separate articles for the stable announcement.

**Pull request merged**